database/models/messages.db.php

```diff
@@ -33,7 +33,7 @@ public function __construct(User $sender, Conversation $conversation, string $co
         $this->sender = $sender;
         $this->conversation = $conversation;
         $this->content = $content;
-        $this->sentAt = new DateTime();
+        $this->sentAt = new DateTime(timezone: new DateTimeZone('Europe/Paris'));
```

Comment:
Don't we have a helper for timezones?

Reply:
nope, but removing this for now. We need to handle timezones properly though